### ISSUE TITLE
(Self-Voicing general) Consider providing role information 

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#250]
At the beginning of the Self-Voicing design process we were not considering full access for non-visual learners. In other  words we were not trying to design an experience equivalent to using screen reader software.

Based on @emily-phet's  notes in [JT & GFLB Notes](https://phetsims.slack.com/files/U6DER46F3/F01DZ4WP22Y/jt-and-gflb-notes.docx), we may want to reconsider if and how we delivery role information.

@emily-phet suggested things like:
- readable for the reading blocks that are not directly changeable
- and roles such as checkbox and button

I think an option to turn on role information is something worth considering more, but I am not sure we need it before the Haptics interviews?

For non-visual access, we have chosen a very straightforward gesture pattern:
1. Double tap - activates buttons an check boxes. This is native behaviour fro VO on iOS
2. Double tap - delivers instructions if the interaction is custom, i.e., our custom grab and drag for sliders and number spinners.

Until we know from learners that they really want to know if an item is a checkbox or a button before they activate it, I am not sure we need to provide role information by default.

Do we want to consider having an option for this?

**Comment by terracoda** [comment]
@Matthew-Moore240, feel free to chime in, and I have added the meeting label so we can discuss more on Wednesday.

**Comment by terracoda** [comment]
After first interview with a person who has low vision but who also often uses a screen reader role information could be useful, but I feel that more context that can be provided through help text might be enough.

We'll see in upcoming interviews this month (March).

**Comment by terracoda** [comment]
Based on interviews, design research, API changes, etc. thus far, I think we are doing pretty well without the need for role information.

I am going to close this issue.
